usagov/vote-gov-drupal
# VOTE-1758: default lang selection for nvrf download

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[VOTE-1758](https://bixal-projects.atlassian.net/browse/VOTE-1758)

## Description (optional)
Adjust NVRF dropdown list of languages so that the selected language is top of the list and selected for download to match the Hugo site.

## Deployment and testing (required, if applicable)
### Pre-deploy steps
1. lando retune

### Testing steps
**Edit: Language codes will need to be added to http://vote-gov.lndo.site/block/3/translations for the dropdown to work properly

1. Go to http://vote-gov.lndo.site/nv/register/ar
2. Confirm Navajo is already selected for NVRF dropdown
3. Switch languages and confirm the current language is now the default for the dropdown
